flink-runtime/src/test/java/org/apache/flink/runtime/minicluster/TestingMiniCluster.java

```diff
@@ -74,8 +74,13 @@ public TestingMiniCluster(TestingMiniClusterConfiguration miniClusterConfigurati
 	}
 
 	@Override
-	public void startTaskExecutor(boolean localCommunication) throws Exception {
```

Comment:
Is it crucial for testing to be able to set the right `localCommunication` flag? If yes, a method overload that sets `localCommunication` to `false` would have been enough but I am not insisting on it.

Reply:
are you suggesting to always start the `TaskExecutors` with `localCommunication = false`? Or only for the `TestingMiniCluster`? The latter should now be the case.

Reply:
Actually, this functionality was needed for #7690.